bluetooth/src/androidLibMain/kotlin/BluetoothMonitor.kt

```diff
@@ -46,26 +49,25 @@ actual class BluetoothMonitor internal constructor(
         }
     }
 
-    private val isPoweredOn: Boolean
+    override val isServiceEnabled: Boolean
         get() = bluetoothAdapter?.isEnabled == true
 
-    private val _isEnabled = MutableStateFlow(isPoweredOn)
-    actual val isEnabled = _isEnabled.asStateFlow()
-
-    actual fun startMonitoring() {
-        if (bluetoothAdapter == null) return
+    override fun startMonitoring() {
+        super.startMonitoring()
         applicationContext.registerReceiver(
             availabilityBroadcastReceiver,
             IntentFilter(BluetoothAdapter.ACTION_STATE_CHANGED)
         )
+        updateEnabledState()
```

Comment:
You have to call `updateState` from abstract class here (and remove updateEnabledState function below)

Reply:
082b88e 64a796b